root/static/scripts/common/components/Autocomplete.js

```diff
@@ -67,7 +67,9 @@ class Autocomplete extends React.Component {
       autocomplete.element.val(next.name);
     }
 
-    if (nextProps.hasOwnProperty('isLookupPerformed')) {
+    if (Object.prototype.hasOwnProperty.call(
+      nextProps, 'isLookupPerformed'
+    )) {
```

Comment:
I would prefer this to be:

```
    if (Object.prototype.hasOwnProperty.call(
      nextProps,
      'isLookupPerformed',
    )) {
```

(same for the other one below on TagEditor.js)